nepalingo-web/src/pages/Home.tsx

```diff
@@ -43,7 +43,12 @@ const Home: React.FC = () => {
           </div>
         </div>
         <div className="mb-5 pt-5">
-          <DailyQuiz />
+          <ActivityCard 
+            backgroundImageUrl="https://t4.ftcdn.net/jpg/06/28/27/29/360_F_628272989_krphWxzqARpTZ20wOBU0YeH2Ecf7iCtI.jpg"
```

Comment:
where did this image come from? can we download and put in our repo? that way if this server goes down the image doesn't break